data/score-cards.json

```diff
@@ -1,1000 +1,1000 @@
 {
```

Comment:
Standardize fieldnames to make it easier to type score card json objects

Reply:
Huzzah to this PR! If you're willing, it would be helpful to split out this change into a dedicated precursor PR so that this PR is smaller, making it easier to review and lower risk.

Reply:
Surely.  See: https://github.com/ParkingReformNetwork/parking-lot-map/pull/179

I'll rebase this and open for review after that's merged.